client/src/components/Ticketing/ticketingmanager/dashboard.tsx

```diff
@@ -50,6 +50,17 @@ const Dashboard = () => {
 
             </div>
           </a>
+          <a onClick={() => navigate('/ticketing/purchaseticket')} className="hover:scale-105 md:h-60 md:w- transition duration-300 ease-in-out border-none  bg-gradient-to-r from-green-500 to-zinc-500 rounded-3xl  shadow-xl  ">
+            <div className="m-8 flex flex-col items-start">
+              <svg xmlns="http://www.w3.org/2000/svg" fill="white" className="bi bi-credit-card h-12 w-12" viewBox="0 0 20 20">
```

Comment:
I noticed that something felt off about the icon you chose, and it looks like it's because the icon you grabbed is from heroicons v2, but we are using heroicons v1: https://v1.heroicons.com/ So you'll probably want to get the credit card icon from here instead.